package/doc/sphinx/Makefile

```diff
@@ -3,7 +3,7 @@
 
 # You can set these variables from the command line.
 SPHINXOPTS    =
-SPHINXBUILD   = sphinx-build
+SPHINXBUILD   = sphinx-build -v -W
```

Comment:
Verbose (more output) and make warnings failure (the same thing that you (?) enabled in the setup.cfg file).

Running 
```
python setup.py build_sphinx
```
is apparently not using this `Makefile` but if one wants to reproduce locally the build with the `Makefile` then I found it useful to have. I didn't know how to change any other `build_sphinx` option for testing so it was convenient to use the standard `Makefile`.